### PR TITLE
Article page remake

### PR DESCRIPTION
**Pull request by andemerie**

### CODE REVIEW

**Comment by UladBohdan** [comment]
@andemerie could you please provide a screenshot?

**Comment by UladBohdan** [comment]
@babajka/developers and don't forget about proper commit naming before pushing!

File: src/article-page/article-page.ejs [hunk 1]
```diff
@@ -4,114 +4,276 @@
...
-            <div class="heading">
-                "Гэта ня тое як склеіць разьбіты кубак..."
+<div class="container">
+    <div class="columns">
+        <div class="column article">
+            <div class="article-top">
+                <h1 class="title is-size-2 has-text-centered article-top__title">
+                    &laquo;Гэта ня тое як склеіць разьбіты кубак...&raquo;
+                </h1>
+                <div class="article-top__sub">
```
Comment (tyndria) [inline src/article-page/article-page.ejs]: I got the BEM methodology and I will try to use it to better understand it advantages :)
But there is one confusing moment: we have article page and we use parent class 'article' in the top of our template. Why do we need use word 'article' futher? Maybe, it's better to omit it and just use top__sub, top__title and etc?

**Comment by andemerie** [comment]
article_page1
article_page2

**Comment by Drapegnik** [comment]
@andemerie I think navbar should be wider than the image

**Comment by UladBohdan** [comment]
@andemerie I like it!

Some notes here are:
* I would like navbar to be wider than the article itself (agree with @Drapegnik ). I'd say navbar should have the same width as navbar on the Main Page (or any other pages)
* There're a lot of elements which we're not going to implement at the moment: bookmarks, read counter, "time-to-read", tags. I guess it's fine to have them in markup although that's how markup is going to differ from page on launch.
* Do we plan to have different tiles on Article page and on Main page? or you were about to replace tiles on Main Page with those you've implemented?
* I'd add more space between tiles and footer.
* What is a "Рубрыка"? A brand? A collection?
* left-right arrows looks a bit awkward. And maybe too small and unnoticeable.
* A weird behavior on mobile devices of: social networks sharing buttons, featuring tiles, the text between the title and the tags on top of the page. And navbar would look better if it uses full width of the screen (mobile screen).

**Comment by Drapegnik** [comment]
I broke github 😭

**Pull request closed without merging**